flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/slot/TaskSlot.java

```diff
@@ -295,22 +303,25 @@ public String toString() {
 				// and set the slot state to releasing so that it gets eventually freed
 				tasks.values().forEach(task -> task.failExternally(cause));
 			}
+
 			final CompletableFuture<Void> cleanupFuture = FutureUtils
```

Comment:
maybe rename this to `shutdownFuture`? We aren't quite sure whether the cleanup was successful.

Reply:
true